pkg/ingester/limiter.go

```diff
@@ -63,6 +85,7 @@ func (l *Limiter) IsWithinMaxMetadataPerMetric(userID string, metadata int) bool
 // number of series in input; otherwise returns false.
 func (l *Limiter) IsWithinMaxSeriesPerUser(userID string, series int) bool {
 	actualLimit := l.maxSeriesPerUser(userID)
+	l.metrics.maxLocalSeriesPerUser.WithLabelValues(userID).Set(float64(actualLimit))
```

Comment:
it feels like hooking up to the runtime config update loop is a more appropriate way to do this. Here's an example https://github.com/grafana/mimir/blob/5e36d6d4b53bd811a8e6a8fc580da2d54f89d46b/pkg/mimir/runtime_config.go#L111-L124

Will this be too complicated?

Reply:
rather than the runtime config update loop, what about another ticker within the [ingester's updateLoop](https://github.com/grafana/mimir/blob/main/pkg/ingester/ingester.go#L520)?

Reply:
i think it fits pretty well there 👍

Reply:
Moved the update logic, which made the overall code simpler too (metrics are now just part of the ingester metrics, rather than a separate limiter metrics block). I don't think this deserves its own config option to specify the update interval, so I just set it at `15s`.